fe2-android/app/src/main/res/values/strings.xml

```diff
@@ -224,16 +224,21 @@
   <string name="digital_cash_beneficiary_address">Beneficiary: address</string>
   <string name="digital_cash_my_address">My Address: address</string>
   <string name="digital_cash_issue_button">Issue</string>
+  <string name="digital_cash_select_0">Individual from the Selector</string>
   <string name="digital_cash_select_1">All LAO members</string>
   <string name="digital_cash_select_2">All LAO attendees</string>
   <string name="digital_cash_select_3">All LAO witnesses</string>
```

Comment:
Maybe have more explicit names so that when we want to add some new ones it is clear which ones are missing. It is a minor detail, it is still understandable as you did it
```suggestion
  <string name="digital_cash_select_single_member">Individual from the Selector</string>
  <string name="digital_cash_select_all_members">All LAO members</string>
  <string name="digital_cash_select_all_attendees">All LAO attendees</string>
  <string name="digital_cash_select_all_witnesses">All LAO witnesses</string>
```